src/supertux/command_line_arguments.hpp

```diff
@@ -76,6 +76,8 @@ class CommandLineArguments
 
   boost::optional<std::string> repository_url;
 
+  boost::optional<std::string> editor_level;
```

Comment:
You should initialize this variable in `src/supertux/command_line_arguments.cpp`, too. See <https://travis-ci.org/SuperTux/supertux/jobs/141392489#L1959-L1963>.